src/main/groovy/com/vanniktech/android/javadoc/Generation.groovy

```diff
@@ -25,29 +29,53 @@ class Generation implements Plugin<Project> {
 
         project.extensions.create("androidJavadoc", AndroidJavadocExtension)
 
-        project.allprojects {
-            afterEvaluate { Project p ->
-                // Do not throw exception anymore. We want to support big projects with a lot of modules that not all of them are android.
-                if (p.hasProperty('android')) {
-                    applyPluginToProject(p);
-                } else {
-                    logger.info "${p.name} is not an android project - plugin is not applied"
+        // Do not apply in root project anymore. User will have to apply plugin manually in all child project.
```

Comment:
I'm actually a fan of having it on the root project and then apply everywhere else.

Why do you want to change it?

Reply:
Ok, you convince me. I've put again the project.allProject{}